make/dpcpp.inc

```diff
@@ -27,6 +27,9 @@ ifeq ($(os_name), linux)
     ifeq ($(use_unnamed_lambda), 1)
         CPLUS_FLAGS += -fsycl-unnamed-lambda
     endif
+    ifeq ($(split_per_kernel), 1)
+        CPLUS_FLAGS += -fsycl-fsycl-device-code-split=per_kernel
```

Comment:
Extra "-fsycl" needs to be removed

Reply:
why is it extra?
its added one time on command line 
`make my_test.pass -B compiler=dpcpp .....`

Reply:
I mean, there is no "-fsycl-fsycl-device-code-split=per_kernel" option. There is "-fsycl-device-code-split=per_kernel" instead.